MyFirstSlideshow/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,11 +4,87 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="com.charles.myfirstslideshow.MainActivity">
```

Comment:
activity_main : line 1 to 6 - Before this layout there should be scrollview should be there.So that view can view on large as well as small resolution devices.

Reply:
activity_main : line 1 to 6- use scroll view to manage the view.